### PR TITLE
OPENEUROPA-1603: Scaffolding files of module.

### PR DESCRIPTION
**Pull request by sergepavle**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        "php": "^7.1"
+    },
+    "require-dev": {
+        "composer/installers": "~1.5",
+        "drupal-composer/drupal-scaffold": "~2.2",
+        "drupal/config_devel": "~1.2",
+        "drupal/console": "~1.0",
+        "drupal/drupal-extension": "~4.0.0@beta",
+        "drush/drush": "~9.0@stable",
+        "nikic/php-parser": "~3.0",
```
Comment (drupol) [inline composer.json]: I don't think this package is necessary.

Comment (sergepavle) [reply composer.json]: I think I remember this package. We had issue with Functional testing within media module. @voidtek proposed add this package (with this version constrain) which fixed problem.
I will remove. if will have problems, we could return.
Thanks!

File: behat.yml.dist [hunk 2]
```diff
@@ -0,0 +1,20 @@
+default:
+  suites:
+    default:
+      paths:
+        - %paths.base%/tests/features
+      contexts:
+        - Drupal\DrupalExtension\Context\MinkContext
+        - Drupal\DrupalExtension\Context\DrupalContext
+  extensions:
+    Behat\MinkExtension:
```
Comment (voidtek) [inline behat.yml.dist]: see the ticket OPENEUROPA-1609.

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    },
+    "require-dev": {
+        "composer/installers": "~1.5",
+        "drupal-composer/drupal-scaffold": "~2.2",
+        "drupal/config_devel": "~1.2",
+        "drupal/console": "~1.0",
+        "drupal/drupal-extension": "~4.0.0@beta",
+        "drush/drush": "~9.0@stable",
+        "openeuropa/code-review": "~1.0.0-alpha4",
+        "openeuropa/drupal-core-require-dev": "^8.6",
```
Comment (voidtek) [inline composer.json]: Should be `"openeuropa/drupal-core-require-dev": "~8.6"`

File: grumphp.yml.dist [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  tasks.phpcs.ignore_patterns:
+    - vendor/
+    - node_modules/
+    - build/
+  tasks.phpcs.triggered_by:
+    - php
+    - module
+    - inc
+    - theme
+    - install
```
Comment (voidtek) [inline grumphp.yml.dist]: we should add `- yml`

File: behat.yml.dist (hunk 2 above)
Comment (sergepavle) [reply behat.yml.dist]: fixed

File: composer.json (hunk 3 above)
Comment (sergepavle) [reply composer.json]: fixed

File: grumphp.yml.dist (hunk 4 above)
Comment (sergepavle) [reply grumphp.yml.dist]: fixed

**Pull request merged**